src/pages/index.astro

```diff
@@ -60,10 +59,10 @@ import { waitlistUrl } from '../waitlist';
       <Hero />
       <StackyIntro />
     </div>
-    <section id="mission" class="comfy">
-      <Mission />
-    </section>
-    <Problems />
+    <!-- <section id="mission" class="comfy"> -->
+    <!--   <Mission /> -->
+    <!-- </section> -->
+    <!-- <Problems /> -->
```

Comment:
I think we should delete unused things :smile: I don't see us bringing this back any time soon – and even if we wanted to, they live in the commit history. Thoughts?